NEWS

```diff
@@ -63,6 +63,8 @@ Build changes
 
 These are compiler, configuration and makefile based changes.
 
+* HTSlib now uses libhtscodecs release 1.1.1
```

Comment:
Fullstop? (or was that intentional as there's so many dots in the last word already :smile:)

Reply:
Kind-of due to the number of dots, but I guess I may as well go for it.